clients/datasetapi.go

```diff
@@ -12,5 +12,6 @@ import (
 // DatasetApiClient defines the zebedee client
 type DatasetClient interface {
 	Checker(context.Context, *healthcheck.CheckState) error
+	GetDatasetCurrentAndNext(ctx context.Context, userAuthToken, serviceAuthToken, collectionID, datasetID string) (m datasetclient.Dataset, err error)
```

Comment:
We shouldn't need this method once we add the additional metadata to the metadata endpoint

Reply:
Add dependency with dataset api PR, and removed latest commit